tests/clims/api/endpoints/test_work_batch_details.py

```diff
@@ -32,6 +35,36 @@ def test_get_by_id(self):
         assert response.data['name'] == 'my_workbatch'
         assert response.data['properties']['kit_type']['value'] == 'kit type value'
 
+    @pytest.mark.dev_edvard
```

Comment:
dev_edvard strikes again!

Reply:
Perhaps we could have a discussion about marks in tests. I actually think it's very handy to keep the marks.

Reply:
Sure! Maybe we can use contextual names for the marks though?